package.json

```diff
@@ -66,6 +63,7 @@
     "@metamask/eslint-config-nodejs": "^12.1.0",
     "@metamask/eslint-config-typescript": "^12.1.0",
     "@metamask/snaps-cli": "^6.0.2",
+    "@metamask/snaps-controllers": "^6.0.4",
```

Comment:
Ehm.. not a fan that we need to have another import just for the Preinstalled snap type.

But at least this is a dev dependency.

Lets keep an eye on this in case it causes issues (circular deps?)